exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +8,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = true; // finish this statement
```

Comment:
Try to use comparison operators in this exercise eg. `studentCount > mentorCount`. You can assign this expression to the `moreStudentsThanMentors` variable:
`let moreStudentsThanMentors = studentCount > mentorCount`
The value of `moreStudentsThanMentors` is `true`

Reply:
Thanks @Simon2828 for reviewing my code.Yes definitely I should use comparison,I just did change in my code.